contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -43,135 +63,109 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
         _erc20Vault = erc20vault_;
         _vault = vault_;
         _marginEngine = IMarginEngine(vault_.marginEngine());
+        _periphery = IPeriphery(vault_.periphery());
+        _vamm = IVAMM(vault_.vamm());
         _currentPosition = vault_.currentPosition();
         _tokens = vault_.vaultTokens();
         _pullExistentials = vault_.pullExistentials();
-    }
 
-    /// @notice Get new signal and act according to it
-    /// @param signal New signal (1 - short/fixed, 2 - long/variable, 3 - exit)
-    /// @param leverage Leverage you take for this trade (in wad)
-    /// @param marginUpdate This flags that this update is triggered by deposit/withdrawal
-    function update(
-        uint256 signal, 
-        uint256 leverage,
-        bool marginUpdate
-    ) public {
+        emit StrategyDeployment(erc20vault_, vault_, admin_);
+    }
+    
+    function setConstants(int24 logProx, uint256 sigmaWad, int256 max_possible_lower_bound_wad, int24 tickSpacing) public {
```

Comment:
I understand if you use this function for tests. However, variables should have their independent setter/getter.

Reply:
Added setters and getters